assets/package.json

```diff
@@ -46,6 +46,7 @@
     "babel-loader": "^8.2.2",
     "css-loader": "^6.4.0",
     "css-minimizer-webpack-plugin": "^3.0.2",
+    "html-loader": "^3.0.1",
```

Comment:
Should we remove vega, vega-embed, and vega-lite?

Reply:
I'd keep it until we release Livebook 0.5.0, so that people running on `main` can still use the current Kino version.